kubectl-plugin/pkg/cmd/version/version.go

```diff
@@ -0,0 +1,20 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var Version = "development"
```

Comment:
```suggestion
const Version string = "development"
```

Reply:
This cannot be replaced with const. This variable will be override by ldflags at build time.